### PR TITLE
added ens script

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
A script that can read data feeds via ens addresses

### CODE REVIEW

**Comment by pappas999** [comment]
code looks good @PatrickAlphaC . Re: the comment that says 
`// This script only works with --network mainnet' `

can we also use this with a local hardhat node running a forked mainnet?

**Comment by PatrickAlphaC** [comment]
Ah good call, yes we should be able to do a forked mainnet

File: tasks/price-consumer/read-price-feed-ens.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// This script only works with '--network mainnet' or 'mainnet-fork'
```
Comment (pappas999) [inline tasks/price-consumer/read-price-feed-ens.js:1]:
```suggestion
// This script only works with --network 'mainnet', or 'hardhat' when running a fork of mainnet 
```

**Comment by pappas999** [comment]
looks good, just made a suggestion to the comment. If that's accepted then the PR can be merged

**Comment by PatrickAlphaC** [comment]
Added your bit in, thanks!

**Pull request merged**